task.sql

```diff
@@ -2,0 +1,38 @@
+CREATE DATABASE ShopDB;
+
+USE ShopDB;
+
+CREATE TABLE Products (
+    ID INT AUTO_INCREMENT,
+    Name VARCHAR(50),
+    Description TEXT,
+    Price INT,
+    WarehouseAmount INT,
+    PRIMARY KEY (ID)
+);
+
+CREATE TABLE Customers (
+    ID INT AUTO_INCREMENT,
+    FirstName VARCHAR(50),
+    LastName VARCHAR(50),
+    Email VARCHAR(50),
+    Address VARCHAR(255),
+    PRIMARY KEY (ID)
+);
+
+CREATE TABLE Orders (
+    OrdersID INT AUTO_INCREMENT,
```

Comment:
The primary key column in the `Orders` table should be renamed from `OrdersID` to `ID` to match the task requirements.